tests/lib/rules/comma-dangle.js

```diff
@@ -567,25 +575,37 @@ ruleTester.run("comma-dangle", rule, {
             ]
         },
         {
-            code: "var foo = {a: 1, b: 2,\nc: 3, d: 4,}",
+            code: "var foo = {\na: 1, b: 2, c: 3, d: 4,\n}",
```

Comment:
Because they weren't failures anymore. The comma is expected in both, since they are multiline.